frontend/components/wallet/WalletDepositWithdrawBlock.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+    <div class="WalletDepositWithDrawBlock">
+        <TextBlock v-if="isExplanationsShown" title="Moving funds">
+            The DAI funds can be moved freely between VAT and your wallet without any extra conversion. You only need
+            to be aware that each transfer incurs transaction fees. Also, if you do not have DAI funds to deposit yet,
+            there are several ways to obtain it:
+            <ul class="list-disc list-inside">
+                <li>By borrowing DAI against a collateral in the oasis.app</li>
+                <li>
+                    By purchasing it on a decentralized exchange like uniswap.org (correct DAI token address used on
+                    the “mainnet” network is
+                    <FormatAddress type="address" value="0x6b175474e89094c44da98b954eedeac495271d0f" shorten />)
```

Comment:
Please don't hardcode the value, but provide it via the prop, ie `tokenAddressDAI`?